### ISSUE TITLE
Logstash 5.0-rc1 RPM packages are not signed

### ISSUE THREAD

**Issue by joshuar** [elastic/logstash#6057]
It looks like the 5.0-rc1 RPM packages are not signed and this causes installation to fail by default.
- Version: 5.0-rc1
- Operating System: CentOS
- Steps to Reproduce:

I'm following the steps [here](https://www.elastic.co/guide/en/logstash/5.0/installing-logstash.html#_yum) on a fresh CentOS 7 install:

```
# after adding the repo file manually
[vagrant@localhost ~]$ sudo rpm --import https://artifacts.elastic.co/GPG-KEY-elasticsearch
[vagrant@localhost ~]$ sudo yum install logstash
Loaded plugins: fastestmirror
logstash-5.x                                                                                                                                                                                                           | 1.3 kB  00:00:00     
logstash-5.x/primary                                                                                                                                                                                                   | 9.1 kB  00:00:01     
Loading mirror speeds from cached hostfile
 * base: centos.mirror.serversaustralia.com.au
 * extras: centos.mirror.serversaustralia.com.au
 * updates: centos.mirror.serversaustralia.com.au
logstash-5.x                                                                                                                                                                                                                            20/20
Resolving Dependencies
--> Running transaction check
---> Package logstash.noarch 1:5.0.0~rc1-1 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

==============================================================================================================================================================================================================================================
 Package                                                 Arch                                                  Version                                                      Repository                                                   Size
==============================================================================================================================================================================================================================================
Installing:
 logstash                                                noarch                                                1:5.0.0~rc1-1                                                logstash-5.x                                                101 M

Transaction Summary
==============================================================================================================================================================================================================================================
Install  1 Package

Total download size: 101 M
Installed size: 189 M
Is this ok [y/d/N]: y
Downloading packages:
Package logstash-5.0.0-rc1.rpm is not signed                                                             99% [=============================================================================================-] 2.5 MB/s | 100 MB  00:00:00 ETA 
logstash-5.0.0-rc1.rpm                                                                                                                                                                                                 | 101 MB  00:00:48     


Package logstash-5.0.0-rc1.rpm is not signed
```

**Comment by suyograo** [comment]
@joshuar ack. this is a known issue with unified release. we are working on it.

**Comment by jordansissel** [comment]
I believe this is solved with 5.0.0 GA
